Needs docker go image to be updated

Docker build is failing: 
```
Step 1/5 : FROM golang:1.16-alpine
  1.1[6](https://github.com/eoch-github/predictions-service/actions/runs/3999215101/jobs/6862833481#step:2:6)-alpine: Pulling from library/golang
  59bf1c3509f3: Pulling fs layer
  666ba61612fd: Pulling fs layer
  8ed8ca486205: Pulling fs layer
  ca4bf8[7](https://github.com/eoch-github/predictions-service/actions/runs/3999215101/jobs/6862833481#step:2:7)e467a: Pulling fs layer
  0435e0963794: Pulling fs layer
  ca4bf[8](https://github.com/eoch-github/predictions-service/actions/runs/3999215101/jobs/6862833481#step:2:8)7e467a: Waiting
  0435e0[9](https://github.com/eoch-github/predictions-service/actions/runs/3999215101/jobs/6862833481#step:2:9)63794: Waiting
  666ba61612fd: Verifying Checksum
  666ba61612fd: Download complete
  8ed8ca486205: Verifying Checksum
  8ed8ca486205: Download complete
  59bf1c3509f3: Verifying Checksum
  59bf1c3509f3: Download complete
  59bf1c3509f3: Pull complete
  0435e0963794: Verifying Checksum
  0435e0963794: Download complete
  666ba61612fd: Pull complete
  8ed8ca486205: Pull complete
  ca4bf87e467a: Verifying Checksum
  ca4bf87e467a: Download complete
  ca4bf87e467a: Pull complete
  0435e0963794: Pull complete
  Digest: sha256:5616dca835fa90ef13a843824ba58394dad356b7d56198fb7c93cbe76d7d67fe
  Status: Downloaded newer image for golang:1.16-alpine
   ---> 7642119cd161
  Step 2/5 : RUN apk --no-cache add bash git git-lfs &&    go get mvdan.cc/gofumpt
   ---> Running in 3d5ab0a76ca0
  fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/main/x86_64/APKINDEX.tar.gz
  fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/community/x86_64/APKINDEX.tar.gz
  (1/11) Installing ncurses-terminfo-base (6.3_p20211120-r1)
  (2/11) Installing ncurses-libs (6.3_p20211120-r1)
  (3/11) Installing readline (8.1.1-r0)
  (4/11) Installing bash (5.1.16-r0)
  Executing bash-5.1.16-r0.post-install
  (5/11) Installing brotli-libs (1.0.9-r5)
  (6/11) Installing nghttp2-libs (1.46.0-r0)
  (7/11) Installing libcurl (7.80.0-r5)
  (8/11) Installing expat (2.5.0-r0)
  (9/11) Installing pcre2 ([10](https://github.com/eoch-github/predictions-service/actions/runs/3999215101/jobs/6862833481#step:2:10).40-r0)
  (10/[11](https://github.com/eoch-github/predictions-service/actions/runs/3999215101/jobs/6862833481#step:2:11)) Installing git (2.34.6-r0)
  (11/11) Installing git-lfs (3.0.2-r2)
  Executing git-lfs-3.0.2-r2.post-install
  Git LFS initialized.
  Executing busybox-1.34.1-r3.trigger
  OK: 31 MiB in 26 packages
  go: downloading mvdan.cc/gofumpt v0.4.0
  go: downloading golang.org/x/sync v0.0.0-20220819030929-7fc1605a5dde
  go: downloading golang.org/x/sys v0.0.0-20220829200755-d48e67d00261
  go: downloading github.com/google/go-cmp v0.5.8
  go: downloading golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
  go: downloading golang.org/x/tools v0.1.[12](https://github.com/eoch-github/predictions-service/actions/runs/3999215101/jobs/6862833481#step:2:12)
  # mvdan.cc/gofumpt/internal/version
  pkg/mod/mvdan.cc/gofumpt@v0.4.0/internal/version/version.go:65:[15](https://github.com/eoch-github/predictions-service/actions/runs/3999215101/jobs/6862833481#step:2:15): undefined: debug.BuildSetting
  pkg/mod/mvdan.cc/gofumpt@v0.4.0/internal/version/version.go:69:7: info.Settings undefined (type *debug.BuildInfo has no field or method Settings)
  pkg/mod/mvdan.cc/gofumpt@v0.4.0/internal/version/version.go:74:30: info.Settings undefined (type *debug.BuildInfo has no field or method Settings)
  note: module requires Go 1.[18](https://github.com/eoch-github/predictions-service/actions/runs/3999215101/jobs/6862833481#step:2:18)
  The command '/bin/sh -c apk --no-cache add bash git git-lfs &&    go get mvdan.cc/gofumpt' returned a non-zero code: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,8 +1,8 @@
-FROM golang:1.16-alpine
+FROM golang:1.19.5-alpine
 
 # TODO: do we need all of these?
 RUN apk --no-cache add bash git git-lfs &&\
-    go get mvdan.cc/gofumpt
+    go install mvdan.cc/gofumpt@latest
 
 ADD entrypoint.sh /entrypoint.sh
 RUN chmod +x /entrypoint.sh

